deploy/examples/cluster.yaml

```diff
@@ -106,6 +106,9 @@ spec:
     #ipFamily: "IPv6"
     # Ceph daemons to listen on both IPv4 and Ipv6 networks
     #dualStack: false
+    #multiClusterService:
```

Comment:
can we add a one-line comment for what it is?

Reply:
added.